### PR TITLE
Proposed change to missing-or-broken-authorization csharp rule

### PR DESCRIPTION
**Pull request by johanlindfors**
This is a proposed change to the missing-or-broken-authorization rule which takes into consideration that the default setting of .NET is to allow anonymous, and that explicitly adding [AllowAnonymous] is a design choice.

Please advise if this is appropriate or if I'm missing something simple with the original rule,

### CODE REVIEW

**Comment by kurt-r2c** [comment]
This change is a more developer-friendly version of this rule and users who wish to flag the AllowAnonymous decorator may write a custom rule if they need one.

**Review by kurt-r2c — APPROVED** [review]

**Comment by kurt-r2c** [comment]
@johanlindfors approved - just need a CLA signature from you.

**Comment by johanlindfors** [comment]
Strange, I've signed the CLA twice now, still no apparent change on the badge.

**Comment by johanlindfors** [comment]
Updated my email to the appropriate one. Sorry for the confusion.

File: csharp/dotnet/security/audit/missing-or-broken-authorization.yaml [hunk 1]
```diff
@@ -15,6 +15,7 @@ rules:
     - 'CWE-862: Missing Authorization'
     cwe2021-top25: true
     cwe2022-top25: true
+    cwe2023-top35: true
```
Comment (philipturnbull) [inline csharp/dotnet/security/audit/missing-or-broken-authorization.yaml]:
```suggestion
    cwe2023-top25: true
```

Looks good @johanlindfors, just a a small typo 😄

**Pull request merged**